typhos/cli.py

```diff
@@ -154,43 +179,92 @@ def create_suite(devices, cfg=None, fake_devices=False):
                 logger.exception("Unable to load Happi entry: %r", device)
         if fake_devices:
             clear_fake_device(device)
-    if loaded_devs or not devices:
-        logger.debug("Creating empty TyphosSuite ...")
-        suite = typhos.TyphosSuite()
-        logger.info("Loading Tools ...")
-        tools = dict(suite.default_tools)
-        for name, tool in tools.items():
-            suite.add_tool(name, tool())
-        if devices:
-            logger.info("Adding devices ...")
-        for device in loaded_devs:
-            try:
-                suite.add_device(device)
-                suite.show_subdisplay(device)
-            except Exception:
-                logger.exception("Unable to add %r to TyphosSuite",
-                                 device.name)
-        return suite
+    return loaded_devs
+
+
+def suite_from_devices(devices):
```

Comment:
`@classmethod` on `TyphosSuite` perhaps?

Reply:
I think you're correct. I'll keep this conversation unresolved until I go back to clean up this file.